Frameworks/UIKit/UITextView.mm

```diff
@@ -403,8 +403,8 @@ - (void)drawRect:(CGRect)rect {
     NSRange range;
     range.location = 0;
     range.length = INT_MAX;
-    [_layoutManager drawBackgroundForGlyphRange:range atPoint:CGPointMake(_textContainerInset.top, _textContainerInset.left)];
-    [_layoutManager drawGlyphsForGlyphRange:range atPoint:CGPointMake(_textContainerInset.top, _textContainerInset.left)];
+    [_layoutManager drawBackgroundForGlyphRange:range atPoint:CGPointMake(rect.origin.x, rect.origin.y)];
+    [_layoutManager drawGlyphsForGlyphRange:range atPoint:CGPointMake(rect.origin.x, rect.origin.y)];
```

Comment:
Sorry if this is a dupe comment; it looks like github ate my original.

What is _textContainerInset used for now?

Technically there is no guarantee that drawRect: gets called with the full bounds of the view (!) as well, so in this case, you might draw the text in a weird location.